### ISSUE TITLE
'unicorn/no-nested-ternary' conflict with Prettier

### ISSUE THREAD

**Issue by yangmingshan** [prettier/eslint-config-prettier#110]
[eslint-plugin-unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn)'s [no-nested-ternary](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-nested-ternary.md) rule is conflict with Prettier. It's added in [v11](https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v11.0.0).

**Comment by lydell** [comment]
Hi!

Can you show a code example where there are conflicts?

**Comment by yangmingshan** [comment]
Unicorn wants
```js
const foo = i > 100 ? 100 : (i < 50 ? 50 : i);
```
Prettier wants
```js
const foo = i > 100 ? 100 : i < 50 ? 50 : i;
```

**Comment by lydell** [comment]
Released in v6.4.0. Thanks for the report!
